### PR TITLE
solved #447 warning for importer

### PR DESCRIPTION
**Pull request by WGierke**
This pull request should solve #447: if someone wants to import a student whose first and last name already exist in the database then a warning is being displayed.

### CODE REVIEW

File: evap/staff/importers.py [hunk 1]
```diff
@@ -165,6 +165,13 @@ def check_user_data_sanity(self):
             except UserProfile.DoesNotExist:
                 # nothing to do here
                 pass
+                
+            try:
+                user_same_name = UserProfile.objects.get(first_name=user_data.first_name, last_name=user_data.last_name)
+                self.warnings.append(u"Warning: The user {} {} already exists.".format(user_same_name.first_name, user_same_name.last_name))
```
Comment (karyon) [inline evap/staff/importers.py]: please wrap the string in _("string goes here"), this tells django to translate it.

also, @janno42 and i discussed this and we think we need the full data of the existing and new user.

something like
"Warning: The existing user {} ({} {} {}, {}) has the same first and last name as the following user from the enrollment data: {} ({} {} {}, {})". you could copypaste that structure from a few lines above. if you have an idea how to not duplicate the code, please do that as well ;)

File: evap/staff/importers.py [hunk 2]
```diff
@@ -140,6 +140,10 @@ def check_user_data_correctness(self):
                 self.errors.append(_(u'Emailaddress {}: Username cannot be empty for non-external users.').format(user_data.email))
                 return # to avoid duplicate errors with validate
             try:
+                user = UserProfile.objects.get(email=user_data.email)
```
Comment (WGierke) [inline evap/staff/importers.py]: Don't exactly know why I added this :D

**Comment by karyon** [comment]
rebased and added the additional check i had in the pipeline. moved to #497

**Comment by karyon** [comment]
i forgot the tahnk you, so: thank you! :)

**Pull request closed without merging**